README.md

```diff
@@ -138,6 +138,21 @@ slackSend(channel: slackResponse.channelId, message: "Update message now", times
 
 This feature requires [botUser](#bot-user-mode) mode.
 
+#### Emoji Reactions
+
+Add an emoji reaction to a previously-sent message like this:
+
+Example:
+
+```groovy
+def slackResponse = slackSend(channel: "emoji-demo", message: "Here is the primary message")
+slackResponse.addReaction("thumbsup")
+```
+
+This may only work reliably in channels (as opposed to private messages) due to [limitations in the Slack API](https://api.slack.com/methods/chat.postMessage) (See "Post to an IM channel").
```

Comment:
document that this doesn't currently work if jenkins has been restarted in between initial message and the reaction, raise an issue if you would like this?